desktop_version/src/Graphics.cpp

```diff
@@ -3422,6 +3422,9 @@ bool Graphics::reloadresources(void)
     grphx.destroy();
     grphx.init();
 
+    gameScreen.isFiltered = !gameScreen.isFiltered;
+    gameScreen.toggleLinearFilter();
```

Comment:
Not a fan of this double-toggle hack. If `gameTexture` needs to be created differently than other textures, then it should be made more explicit than this.